src/heretic/p_local.h

```diff
@@ -241,8 +241,8 @@ void P_RadiusAttack(mobj_t * spot, mobj_t * source, int damage);
 // ***** P_SETUP *****
 
 extern byte *rejectmatrix;      // for fast sight rejection
-extern short *blockmaplump;     // offsets in blockmap are from here
-extern short *blockmap;
+extern long* blockmaplump;	    // offsets in blockmap are from here
```

Comment:
Please avoid the use of type `long`, it means different things on different platforms. Also, please keep the `type *name` nomenclature for pointers. In general, you should make sure the code you are going to introduce in Crispy Heretic today looks like the pendant code in Crispy Doom *today*, not at the time of the initial commit.

Reply:
Makes sense. I of course was just using the original commit as the guide for the patch, but I can bring it into line with the current state of Crispy Doom.